README.md

```diff
@@ -12,6 +12,10 @@ Type: `label`
 - `addToProject` - an object that is required when the `action` is `addToProject` and is optional otherwise.
 - `addToProject.url`: Absolute url of the project, the project `id` will be parsed.
 - `addToProject.column`: Column name to add the issues to, required for old type of projects
+- `removeFromProject` - an object that is required when the `action` is `removeFromProject` and is optional otherwise.
```

Comment:
maybe state somewhere that, when removed, the issues will irreversibly loose the project-specific metadata assigned to it.